.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -13,17 +14,10 @@ Note: these links work by overwriting query parameters of the current url. If th
 
 _These tasks must be completed to release._
 
-- [ ] Create and request approval of a [Verification Report on Cognidox](https://ultrahaptics.cdox.net/cgi-perl/browse-categories?id=973) for the relevant product being released
-- [ ] Ensure documentation requirements are met e.g., public API is commented
-- [ ] Consider any licensing/other legal implications e.g., notices required
-- [ ] Check that any additional release tasks from contributing PRs have been considered below
-- [ ] Close off the JIRA release
-
-## Additional Release Tasks
-
-_This task list should be populated from PRs contributing to this release. Can include functionality tests and regression tests such as tests for integration of multiple features as well as any other tasks that should be performed during the release._
-
-- [ ] 
+- [ ] Create and request approval of a [Request to Release on Cognidox](https://ultrahaptics.cdox.net/cgi-perl/browse-categories?id=973) for the relevant product being released. This will include several tasks within.
+- [ ] Ensure version numbers within software have been updated
+- [ ] Ensure any changelogs have been updated
```

Comment:
I think we still want the licensing note here

Reply:
It's in the request to release document already so didn't want to duplicate it here. I've left this step on the other templates though.